src/c++/perf_analyzer/genai-pa/tests/test_cli.py

```diff
@@ -24,14 +24,79 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+import io
+import sys
+
 import pytest
 from genai_pa.main import run
 
 
-# NOTE: Placeholder
-class TestHelp:
-    @pytest.mark.parametrize("arg", ["-h", "--help"])
-    def test_help(self, arg):
-        args = [arg]
-        with pytest.raises(SystemExit):
-            run(args)
+class TestCLIArguments:
+    @pytest.mark.parametrize(
+        "arg, expected_output",
+        [
+            (["-h"], "CLI to profile LLMs and Generative AI models with Perf Analyzer"),
+            (
+                ["--help"],
+                "CLI to profile LLMs and Generative AI models with Perf Analyzer",
+            ),
+        ],
+    )
+    def test_help_arguments_output_and_exit(self, arg, expected_output, capsys):
+        combined_args = ["-m", "test_model"] + arg
```

Comment:
Won't this repeat the model name twice on the command line?
The wrapper hard codes `perf_analyzer -m {model} `.

Also, the args passed into run are expected to be a dict. I would expect this to fail the tuple deconstruction if not beforehand.

Reply:
I don't think so. It will fail:

```
self = ArgumentParser(prog='genai-pa', usage=None, description='CLI to profile LLMs and Generative AI models with Perf Analyzer', formatter_class=<class 'argparse.HelpFormatter'>, conflict_handler='error', add_help=True)
status = 2, message = 'genai-pa: error: the following arguments are required: -m/--model\n'
```

I ran all of these unit tests. They pass only with `-m {model}` and have no issues with this construction.

Reply:
Yup. You are totally right.